cirq-google/cirq_google/ops/internal_tag.py

```diff
@@ -57,3 +60,22 @@ def _value_equality_values_(self):
         except TypeError:
             tag_args_eq_values = self.tag_args
         return (self.name, self.package, tag_args_eq_values)
+
+    def to_proto(self, msg: Optional[program_pb2.Tag] = None) -> program_pb2.Tag:
+        if msg is None:
+            msg = program_pb2.Tag()
+        msg.internal_tag.tag_name = self.name
+        msg.internal_tag.tag_package = self.package
+        # for k, v in self.tag_args.items():
```

Comment:
I guess this is left for a follow-up PR?

Reply:
Yes, removed.  Probably needs a refactor since it introduces a circular dependency.